lib/sql_contacts_repository.rb

```diff
@@ -0,0 +1,29 @@
+require 'contacts_db'
+
+class SQLContactsRepository
+
+  def add(contact)
+    Contacts.create(:first_name => contact.first_name,
```

Comment:
Where is this Contacts class defined?  I am no expert with ActiveRecord so we should probably go through it together tomorrow